api/checks/api.go

```diff
@@ -172,6 +179,61 @@ func (s checksAPIImpl) CreateWPTCheckSuite(appID, installationID int64, sha stri
 	return suite != nil, err
 }
 
+func (s checksAPIImpl) GetAzureArtifactsURL(owner, repo string, buildID int64) string {
```

Comment:
I'm confused by `golint`. Why doesn't it complain the lack of comment for this exported method?

Reply:
Not sure either.